tools/myst/myst.edl

```diff
@@ -159,8 +159,7 @@ enclave
             int epfd,
             [out, count=maxevents] struct epoll_event* events,
             size_t maxevents,
-            int timeout)
-            transition_using_threads;
+            int timeout);
```

Comment:
Did this cause any performance issues?

Reply:
There does seem to be a performance impact on solutions/redis benchmark.